pkg/apiclient/offline-client.go

```diff
@@ -0,0 +1,46 @@
+package apiclient
```

Comment:
I liked the implementation. But offline is going to be supported only on `Workflow` lint command. Do we really need to create another client type instead of having a minimized validation function?

Reply:
I know it seems excessive just for lint. Another solution is that we don't support offline (abort this work). Thoughts?

Reply:
This is an existing feature and some users may use this feature like Kubeflow.  I would recommend having this feature with limitations like it won't validate the `TemplatesRef` instead of removing.